### DISCUSSION TITLE
Cannot render <picture> in markdown files

### DISCUSSION THREAD

**Discussion by ameshkov** [community/community#44314] [Mobile]
Steps to reproduce:
1. Open https://github.com/AdguardTeam/AdGuardHome/blob/master/README.md
2. The logo will be broken.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.95.0 (88456997)
Device: iPad Pro 11-inch 2 (iOS 16.1.1)

Screenshot:
![telegram-cloud-photo-size-4-5920443896360451284-y](https://user-images.githubusercontent.com/5947035/212634165-b8eac2b0-90f7-43b5-88fe-594547cdbc5a.jpg)
</details>

**Comment by aliazmoodehvaldi** [comment]
I saw the logo without any problems

**Reply by ameshkov** [reply]
Maybe an iPad app issue only?

I've added the screenshot to Additional debug info in my original post.

**Reply by aliazmoodehvaldi** [reply]
Did you try show image with this ?

! [img] (doc/adguard_home_darkmode.svg)

**Reply by scripthunter7** [reply]
@ameshkov I submitted this logo PR to AG Home, but I have never experienced such an issue, even though I use the picture tag elsewhere. I also think it could be an Apple-related / Apple-specific problem

**Comment by alexeytatarchuk** [comment]
It's displayed now without any problems

**Comment by Jay-o-Way** [comment]
Can reproduce in latest app on Android.
![Screenshot_20230311-124147.png](https://user-images.githubusercontent.com/65828559/224482464-f6283cac-fc35-4723-9ed0-031741f0b8f3.png)

Having that said...
- Why on earth are you writing a MD file as if it were HTML? Make a choice already.
- What happens if you use _only_ \<img> ?

**Reply by Cryptomonkey1979** [reply]
I understand

**Reply by Cryptomonkey1979** [reply]
And I really needed this

**Reply by Cryptomonkey1979** [reply]
I don't know

**Reply by Cryptomonkey1979** [reply]
I hope not because I have an iPhone the person that's hacking. My account has an android.

**Comment by dpuga1** [comment]
I didn’t write that file if it’s an android I do not have an android. I
have an iPhone 15 so you know if it’s an android it is not mine.

310) 713-2784 David Puga Jr

**Comment by dpuga1** [comment]
Yes, I really need it. All this hacking and cloning is destroying my life
so stressed out.

310) 713-2784 David Puga Jr

**Comment by dpuga1** [comment]
How can I help you?
I don’t even know the proper terminology for the comments

**Comment by Jay-o-Way** [comment]
@ameshkov the issue seems to be resolved in the meantime. Image renders. Do agree with other comments that the amount of HTML should be kept at a very minimum. Recommend to refurbish the MD file.

**Reply by Cryptomonkey1979** [reply]
I agree thank you everyone for your input and participation

**Reply by Cryptomonkey1979** [reply]
I'm really new to this and I really don't even understand. I'm trying to learn though.

**Comment by dpuga1** [comment]
Thanks for the feedback!

**Comment by dpuga1** [comment]
Thank you, thank you thank you I will help you.

**Comment by dpuga1** [comment]
The problem is, I don’t know how to keep them out of my email. They’ve
taken it over completely my account like they turned it into a group
account. It’s a personal email account. That’s what I created in 1999.
That’s when I opened it up. I never turned it into a group email account
and I don’t know how to fix it.

310) 713-2784 David Puga Jr
